AppCenterDataStorage/AppCenterDataStorageTests/Fixtures/invalidTestDocument.json

```
@@ -0,0 +1,12 @@
+{
+    "_self" : "dbs/HqsVAA==/colls/HqsVAMg0hU4=/docs/HqsVAMg0hU4EAAAAAAAAAA==/",
+    "id" : "standalonedocument1",
+    "_attachments" : "attachments/",
+    "document" : {
+        1 : "invalid index"
```

GitHub UI highlights invalid format of JSON file.
```suggestion
        "1" : "invalid index"
```